tools/mtmd/clip.cpp

```diff
@@ -1891,6 +1891,18 @@ struct clip_graph {
             cur = ggml_gelu_erf(ctx0, cur);
             cur = ggml_mul_mat(ctx0, model.mm_2_w, cur);
 
+        } else if (ctx->proj_type() == PROJECTOR_TYPE_GLMA) {
+            cur = ggml_norm(ctx0, cur, hparams.eps);
+            cur = ggml_mul(ctx0, cur, model.mm_norm_pre_w);
+            cur = ggml_add(ctx0, cur, model.mm_norm_pre_b);
+            cur = ggml_reshape_2d(ctx0, cur, cur->ne[0] * 4, cur->ne[1] / 4);
```

Comment:
this will fail if number of elements is not divisible by 4

instead, you should abstract out the `StackAudioFrames` used by ultravox into a new function, `build_stack()`, and reuse it here